lib/sendLineNotify.ts

```diff
@@ -2,7 +2,7 @@ import { got } from "got";
 
 export async function sendLineNotify(
   messageContent: string,
-  msgImg: string,
+  msgImg: string | undefined,
   lineNotifyToken: string | undefined
```

Comment:
Maybe we should drop the `lineNotifyToken` parameter and use `config.lineToken` directly inside the method